vpr/src/place/place.cpp

```diff
@@ -745,15 +758,20 @@ void try_place(const t_placer_opts& placer_opts,
 
     first_rlim = (float)max(device_ctx.grid.width() - 1, device_ctx.grid.height() - 1);
 
-    float first_t = starting_t(&costs, &prev_inverse_costs,
-                               annealing_sched, move_lim, first_rlim,
-                               place_delay_model.get(),
-                               placer_criticalities.get(),
-                               timing_info.get(),
-                               *move_generator,
-                               pin_timing_invalidator.get(),
-                               blocks_affected,
-                               placer_opts);
+    float first_t = 0;
+#ifdef ENABLE_ANALYTIC_PLACE
+    // Analytic placer: When enabled, disables most of the anneal by setting first_t = 0 and goes strait to quench
+    if (!placer_opts.enable_analytic_placer)
+#endif /* ENABLE_ANALYTIC_PLACE */
+        first_t = starting_t(&costs, &prev_inverse_costs,
+                             annealing_sched, move_lim, first_rlim,
+                             place_delay_model.get(),
+                             placer_criticalities.get(),
+                             timing_info.get(),
+                             *move_generator,
+                             pin_timing_invalidator.get(),
+                             blocks_affected,
+                             placer_opts);
```

Comment:
It seems like if you are using the analytic placer, then the starting temperature for the annealing is 0. Is your intention jumping straight to the placement quench part (1X placement inner loop), or perform the usual annealing (**huge number**X placement inner loop), albeit with a temperature of 0?

Reply:
I'll checkout the conflicts soon. My reason for setting start_t to 0 is to
go straight to quench.